src/components/screens/IntegrationsCatalog/RecipesDetailScreen/CodeSnippets.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect } from 'react';
```

Comment:
Looks like a lot of overlap with https://github.com/storybookjs/frontpage/blob/main/src/components/screens/DocsScreen/CodeSnippets.tsx. Is this one meaningfully different? Could the existing one be moved to `/src/components/basics` and shared in both screens?